.github/workflows/codespell-private.yml

```diff
@@ -4,6 +4,40 @@
 name: codespell
 on: [push, pull_request]
 jobs:
+  test:
+    env:
+      REQUIRE_ASPELL: true
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        python-version:
+          - 3.5
+          - 3.6
+          - 3.7
+          - 3.8
+          - 3.9
+    name: Python ${{ matrix.python-version }} test
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - run: sudo apt-get install libaspell-dev aspell-en
+      - run: |
+          python --version  # just to check
+          pip install -U pip wheel # upgrade to latest pip find 3.5 wheels; wheel to avoid errors
+          pip install codecov chardet "setuptools!=47.2.0" docutils
+          pip install aspell-python-py3
+          pip install -e ".[dev]" # install the codespell dev packages
+      - run: python setup.py install
+      - run: codespell --help
+      - run: make check
+      - run: codespell --check-filenames --skip="./.git/*,*.pyc,./codespell_lib/tests/test_basic.py,./codespell_lib/data/*,./example/code.c,./build/lib/codespell_lib/tests/test_basic.py,./build/lib/codespell_lib/data/*"
+      # this file has an error
+      - run: "! codespell codespell_lib/tests/test_basic.py"
+      - run: codecov
+    
   make-check-dictionaries:
```

Comment:
I've left this separate so we only comment on any issues once rather than once per language generating a lot of noise.